google-cloud-spanner/src/main/java/com/google/cloud/spanner/Value.java

```diff
@@ -2976,6 +3033,8 @@ private Value getValue(int fieldIndex) {
                 return Value.pgNumericArray(value.getStringList(fieldIndex));
               case DATE:
                 return Value.dateArray(value.getDateList(fieldIndex));
+              case UUID:
+                return Value.uuidArray(value.getUuidList(fieldIndex));
```

Comment:
nit: indentation not consistent

Reply:
How?

Reply:
Realised , this is in  case ARRAY scope and hence the tab.  Can be resolved.